docs/advanced/settings.md

```diff
@@ -180,38 +178,42 @@ Default:
 }
 ```
 
-1. Add the provisioner for the newly added CSI driver.
-1. Configure **Volume Snapshot Class Name**, which refers to the name of the `VolumeSnapshotClass` used to create volume snapshots or VM snapshots.
-1. Configure **Backup Volume Snapshot Class Name**, which refers to the name of the `VolumeSnapshotClass` used to create VM backups.
+### `default-vm-termination-grace-period-seconds`
+
+**Versions**: v1.2.0 and later
 
-## `default-vm-termination-grace-period-seconds`
+**Definition**: Number of seconds Harvester waits before stopping a VM.
```

Comment:
@jillian-maroket The idea about this setting is:
- Once a user stops the VM from Harvester GUI, a graceful shutdown signal is sent to the VM.
- If the VM is not shut down in the value defined in the setting `default-vm-termination-grace-period-seconds`, Harvester will shutdown the VM forcibly.

Can you help rephrase the idea? Thanks

Reply:
Thanks for clarifying. Here is the updated description:

**Definition**: Number of seconds Harvester waits before forcibly shutting down a VM that was stopped using the Harvester UI.

Harvester sends a graceful shutdown signal to any VM that is stopped using the Harvester UI. If the graceful shutdown process is not completed within the specified number of seconds, Harvester forcibly shuts down the VM.